random.cabal

```diff
@@ -65,6 +65,7 @@ test-suite doctests
     build-depends:
         base -any,
         doctest >=0.15,
+        mwc-random -any,
```

Comment:
Why?

Reply:
So we can have an example `MonadRandom` implementation: https://github.com/idontgetoutmuch/random/pull/61/files#diff-4a522d098452ebfc9be2d6291588f5bcR279-R295

Perhaps that's just me being lazy though. We could implement our own monadic pseudo-random number generator as an example and then implement `MonadRandom` for it, rather than pulling in `mwc-random` as a dependency.

Reply:
Oh sorry - it's in a test stanza - that's fine